servers/physics_3d/joints/generic_6dof_joint_3d_sw.h

```diff
@@ -65,43 +65,28 @@ class G6DOFRotationalLimitMotor3DSW {
 public:
 	//! limit_parameters
 	//!@{
-	real_t m_loLimit; //!< joint limit
-	real_t m_hiLimit; //!< joint limit
-	real_t m_targetVelocity; //!< target motor velocity
-	real_t m_maxMotorForce; //!< max force on motor
-	real_t m_maxLimitForce; //!< max force on limit
-	real_t m_damping; //!< Damping.
-	real_t m_limitSoftness; //! Relaxation factor
-	real_t m_ERP; //!< Error tolerance factor when joint is at limit
-	real_t m_bounce; //!< restitution factor
-	bool m_enableMotor;
-	bool m_enableLimit;
+	real_t m_loLimit = -1e30; //!< joint limit
+	real_t m_hiLimit = 1e30; //!< joint limit
+	real_t m_targetVelocity = 0.0; //!< target motor velocity
+	real_t m_maxMotorForce = 0.1f; //!< max force on motor
+	real_t m_maxLimitForce = 300.0f; //!< max force on limit
+	real_t m_damping = 1.0f; //!< Damping.
+	real_t m_limitSoftness = 0.5f; //! Relaxation factor
+	real_t m_ERP = 0.5f; //!< Error tolerance factor when joint is at limit
+	real_t m_bounce = 0.0f; //!< restitution factor
```

Comment:
The `f` should be removed since it's `real_t`, so we shouldn't enforce `float` literals.